src/pages/Login/Login.jsx

```diff
@@ -75,93 +74,99 @@ function Login() {
         alignItems="center"
       >
         <Flex direction="column" alignItems="center">
-          <Box 
-            bg={colors.boxColor} 
-            p={10} 
-            borderRadius="md" 
-            width="35vw" 
+          <Box
+            bg={colors.boxColor}
+            p={10}
+            borderRadius="md"
+            width="35vw"
             minHeight="65vh"
           >
             <Flex justifyContent="flex-end">
               <Button
                 mr={2}
-                onClick={handleToggleColorMode}
+                onClick={toggleColorMode}
                 color={colors.buttonColor}
                 backgroundColor={colors.buttonBgColor}
               >
-                Toggle {colors.text} Mode
+                {colorMode === "light" ? <MoonIcon /> : <SunIcon />}
```

Comment:
I'm about to add the icons onto the util file so you can just pass it onto here once i push or merge it